velox/exec/RowContainer.h

```diff
@@ -425,6 +436,14 @@ class RowContainer {
     return 1 << (nullOffset & 7);
   }
 
+  static inline int32_t initializedByte(int32_t nullOffset) {
```

Comment:
What's the new layout? Thanks!

Reply:
Updated the comment describing the layout in the RowContainer constructor, also added comments here.